co2eq/car/index.js

```diff
@@ -19,14 +19,14 @@ export function modelCanRun(activity) {
 }
 
 // look up carbonIntensity by brand name
-export function carbonIntensityByBrand(brand) {
-  const entry = cars.footprints.find(d => d.brand === brand);
-  if (!entry) {
-    throw new Error(`Unknown brand name ${brand}`);
-  } else {
-    return entry.carbonIntensity;
-  }
-}
+// function carbonIntensityByBrand(brand) {
```

Comment:
commented code?

Reply:
I wasn't sure if this is something we need or plan to do soon, otherwise we can kill it?

Reply:
@martincollignon can maybe shed some light here?
Else I don't see any reason to keep it.